lxd/storage/drivers/utils.go

```diff
@@ -573,3 +575,26 @@ func copyDevice(inputPath, outputPath string) error {
 func loopFilePath(poolName string) string {
 	return filepath.Join(shared.VarPath("disks"), fmt.Sprintf("%s.img", poolName))
 }
+
+var errNonGPTDisk = fmt.Errorf("Non-GPT disk")
+
+// moveGPTAltHeader moves the GPT alternative header to the end of the disk device supplied.
+// If the device supplied is not detected as not being a GPT disk then no action is taken and nil is returned.
+func moveGPTAltHeader(devPath string) error {
+	_, err := shared.RunCommand("sgdisk", "--move-second-header", devPath)
```

Comment:
How about only doing this if sgdisk can be found and logging a warning if not?

Not sure how common that tool is.

Reply:
Yep will do.